src/main/java/de/tum/in/www1/artemis/service/messaging/DistributedInstanceMessageSendService.java

```diff
@@ -151,7 +151,13 @@ public void sendAssessedExerciseSubmissionNotificationSchedule(Long exerciseId)
     }
 
     @Override
-    public void sendExamWorkingTimeChangeDuringConduction(Long studentExamId) {
+    public void sendExamWorkingTimeChangeDuringConduction(Long examId) {
+        log.info("Sending schedule to reschedule exam {} to broker.", examId);
+        sendMessageDelayed(MessageTopic.EXAM_RESCHEDULE_DURING_CONDUCTION, examId);
+    }
+
+    @Override
+    public void sendStudentExamWorkingTimeChangeDuringConduction(Long studentExamId) {
```

Comment:
The name of this method looks quite similar to the one above, but they do completely different things. Can you maybe name this one `sendStudentExamIndividualWorkingTimeChangeDuringConduction` to increase code readability? Same for the other methods like `processStudentExamWorkingTimeChangeDuringConduction`.